enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.ts

```diff
@@ -21,7 +21,8 @@ import { SDK_TO_MAIN_APP_COLORS_MAPPING } from "./embedding-color-palette";
 const getFontFamily = (theme: MetabaseTheme) =>
   theme.fontFamily ?? DEFAULT_FONT;
 
-const SDK_BASE_FONT_SIZE = `${DEFAULT_SDK_FONT_SIZE / 16}em`;
+// const SDK_BASE_FONT_SIZE = `${DEFAULT_SDK_FONT_SIZE / 16}em`;
+const SDK_BASE_FONT_SIZE = `${DEFAULT_SDK_FONT_SIZE}px`;
```

Comment:
I don't think the default font size of the sdk should be in rem, otherwise it will be influenced by the font size of the host app.
Some people set the font size to 10px so that they can use 1.6em to mean 16px, I don't think it would work nice with our app.
I think we should use a fixed font size, and if the customer explicitly wants to change the font size then they can use the theme.

Reply:
@npretto I recalled this is more of a **reference value** to [allow font size to scale in all viz components](https://github.com/metabase/metabase/pull/43264) though. It's not what determines the font size. If you use `px` here, then the viz elements' typography will no longer scale correctly.

Reply:
Mmm let me check. Maybe we should put the font size in px in the container but use em inside so that the ems are relative to the px value 🤔.
The whole em/rem thing has always been confusing to me

Reply:
@heypoom is this the expected behaviour?

https://github.com/user-attachments/assets/f58417fa-d9a1-4f55-b60d-570e957b7362

Reply:
@npretto For the Bar Charts, yes that looks good. The trend chart's elements is also supposed to scale though if I remembered correctly, so that is a little weird. Let me check in production if trend charts scale properly in master.

Reply:
@heypoom it loooks like that font is expressed in `rem`, and it seems like if people mess with the font-size on the html tag it still breaks everything in the sdk, but i don't think we can do much about it